# Goal
int32 detect

---
# Result
string distance_reached

---
# Feedback
string reached